src/config/dynamic-theme-fixes.config

```diff
@@ -3328,6 +3328,20 @@ INVERT
 
 ================================
 
+kraftfuttermischwerk.de
+
+INVERT
+.title-container img.logo
+img[alt="Bildschirmfoto 2014-05-02 um 21.18.06"]
```

Comment:
Doesn't look good. It's a image with a fixed background color, shouldn't be touched.